Modules/QtWidgets/src/QmitkIOUtil.cpp

```diff
@@ -245,6 +279,17 @@ QStringList QmitkIOUtil::Save(const std::vector<const mitk::BaseData *> &data,
     // Ask the user for a file name
     QString nextName = QFileDialog::getSaveFileName(parent, dialogTitle, fileName, filterString, &selectedFilter);
 
+    if (IsIllegalFilename(nextName))
+    {
+      QMessageBox::warning(
+        parent,
+        "Saving not possible",
+        QString("File \"%2\" contains invalid characters.\n\nPlease avoid any of %1")
```

Comment:
Would like to see the char list quoted like the filename and a hint towards the whitespace thing you also implemented above.

```suggestion
        QString("File \"%2\" contains invalid characters or leading/trailing whitespace characters.\n\nPlease avoid any of \"%1\".")
```

Reply:
Implemented, see above.